modular_skyrat/master_files/code/modules/awaymission/mission_code/black_mesa/mesa_mobs.dm

```diff
@@ -557,9 +559,22 @@
 
 /obj/effect/random_mob_placer/Initialize(mapload)
 	. = ..()
+	for(var/turf/iterating_turf in range(MOB_PLACER_RANGE, src))
+		RegisterSignal(iterating_turf, COMSIG_ATOM_ENTERED, .proc/trigger)
+
+/obj/effect/random_mob_placer/proc/trigger(datum/source, atom/movable/entered_atom)
+	if(!isliving(entered_atom))
```

Comment:
You should probably add `SIGNAL_HANDLER` on the first line of the proc, as that's typically what's done for procs that are activated by a signal, if I remember correctly?

Reply:
Yee